x/ecocredit/client/testsuite/query.go

```diff
@@ -470,3 +471,17 @@ func (s *IntegrationTestSuite) TestQueryCreditTypes() {
 		})
 	}
 }
+
+func (s *IntegrationTestSuite) TestQueryParams() {
+	val := s.network.Validators[0]
+	clientCtx := val.ClientCtx
+	clientCtx.OutputFormat = "JSON"
+	require := s.Require()
+
+	cmd := client.QueryParams()
+	out, err := cli.ExecTestCLICmd(clientCtx, cmd, []string{})
+	require.NoError(err)
+
+	expected := `{"params":{"credit_class_fee":[{"denom":"stake","amount":"20000000"}],"allowed_class_creators":[],"allowlist_enabled":false,"credit_types":[{"name":"carbon","abbreviation":"C","unit":"metric ton CO2 equivalent","precision":6}]}}`
```

Comment:
if we change the default params, this test will break which could be a pain later down the road. 

Not the prettiest solution, but was able to avoid this with the following code. theres prob a better way but this worked for me 🤷🏻 :

```go
type mockParams struct {
	ecocredit.Params `json:"params"`
}

func (s *IntegrationTestSuite) TestQueryParams() {
	val := s.network.Validators[0]
	clientCtx := val.ClientCtx
	clientCtx.OutputFormat = "JSON"
	require := s.Require()

	cmd := client.QueryParams()
	out, err := cli.ExecTestCLICmd(clientCtx, cmd, []string{})
	require.NoError(err)

	var params mockParams
	json.Unmarshal(out.Bytes(), &params)

	exp := ecocredit.DefaultParams()

	require.Equal(exp, params.Params)
}
```